craco.config.cjs

```diff
@@ -50,23 +51,34 @@ module.exports = {
   webpack: {
     plugins: [new VanillaExtractPlugin({ identifiers: 'short' })],
     configure: (webpackConfig) => {
-      webpackConfig.plugins = webpackConfig.plugins.map((plugin) => {
-        // Extend process.env with dynamic values (eg commit hash).
-        // This will make dynamic values available to JavaScript only, not to interpolated HTML (ie index.html).
-        if (plugin instanceof DefinePlugin) {
-          Object.assign(plugin.definitions['process.env'], {
-            REACT_APP_GIT_COMMIT_HASH: JSON.stringify(commitHash),
-          })
-        }
+      webpackConfig.plugins = webpackConfig.plugins
+        .map((plugin) => {
+          // Extend process.env with dynamic values (eg commit hash).
+          // This will make dynamic values available to JavaScript only, not to interpolated HTML (ie index.html).
+          if (plugin instanceof DefinePlugin) {
+            Object.assign(plugin.definitions['process.env'], {
+              REACT_APP_GIT_COMMIT_HASH: JSON.stringify(commitHash),
+            })
+          }
 
-        // CSS ordering is mitigated through scoping / naming conventions, so we can ignore order warnings.
-        // See https://webpack.js.org/plugins/mini-css-extract-plugin/#remove-order-warnings.
-        if (plugin instanceof MiniCssExtractPlugin) {
-          plugin.options.ignoreOrder = true
-        }
+          // CSS ordering is mitigated through scoping / naming conventions, so we can ignore order warnings.
+          // See https://webpack.js.org/plugins/mini-css-extract-plugin/#remove-order-warnings.
+          if (plugin instanceof MiniCssExtractPlugin) {
+            plugin.options.ignoreOrder = true
+          }
 
-        return plugin
-      })
+          return plugin
+        })
+        .filter((plugin) => {
+          // Case sensitive paths are enforced by typescript.
+          // See https://www.typescriptlang.org/tsconfig#forceConsistentCasingInFileNames.
```

Comment:
nit: would it make sense to make it an array and loop over instead of `if conditions` one below another? Sounds like more extendable in the future, but it's pure stylistic choice.

Reply:
this _is_ a clickable link in VSCode - it renders all http(s) links as clickable. It's not a doc-comment, it's an implementation comment, so I don't want to use JSDoc.

I think this is stylistically easiest for me to read. I had it using `.find` earlier but this makes it easier in case someone wants to modify an additional plugin.